### PR TITLE
Adding Segment heartbeat

### PR DESCRIPTION
**Pull request by adamgordonbell**
Send a heartbeat event to segment every 5 seconds when tab / window is active for purposes of tracking session duration in our analytics.

### CODE REVIEW

File: blog/_includes/head/growthbook.html [hunk 1]
```diff
@@ -20,23 +20,4 @@
       });
     }
   });
-
```
Comment (adamgordonbell) [inline blog/_includes/head/growthbook.html:23]: Removing bolding experiment

**Review by vladaionescu — COMMENTED** [review]
Could this increase our Segment bill?

**Comment by adamgordonbell** [comment]
> Could this increase our Segment bill?

@thtmnisamnstr may know how it impacts that. But, we are generating more events for sure.

**Comment by thtmnisamnstr** [comment]
@adamgordonbell The pattern we generally follow for this type of thing is to create a new `.js` file in /[website/blog]/assets/js/ (e.g. /blog/assets/js/analytics.js) with the applicable Javascript and then add it in the /[website/blog]/_includes/head.html (e.g. `<script src="/blog/assets/js/analytics.js"></script>`).

**Comment by adamgordonbell** [comment]
> @adamgordonbell The pattern we generally follow for this type of thing is to create a new `.js` file in /[website/blog]/assets/js/ (e.g. /blog/assets/js/analytics.js) with the applicable Javascript and then add it in the /[website/blog]/_includes/head.html (e.g. `<script src="/blog/assets/js/analytics.js"></script>`).

Done, @thtmnisamnstr Take a look.

**Comment by thtmnisamnstr** [comment]
LGTM

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**